### DISCUSSION TITLE
Cannot Signup

### DISCUSSION THREAD

**Discussion by EvanDevvv** [vercel/community#3283] [Help] [answered]
### Summary

So I went to signup with github, and I did get an email saying "Welcome to Vercel!", but when I go to login, it says this user is blocked. When I go to signin with my email it says "We are unable to complete your signup."
![Annotation 2023-07-18 015917](https://github.com/vercel/community/assets/123332223/f5e27cbd-d1ec-46ad-b4c7-783d71102f4b)
![Annotation 2023-07-18 015938](https://github.com/vercel/community/assets/123332223/b3d366c6-5b29-4d0e-902a-cae50cd3526a)

### Steps to Reproduce

Signup or Login.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
